src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -39,4 +55,29 @@ public function getResponse() {
     return $this->response;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function missingDependencies() {
+    $capture_message = $this->t('Screen Capture package missing.');
```

Comment:
It didn't appear to be, which is why I added `use StringTranslationTrait;` to the base capture utility class.